Source/Core/AudioCommon/OpenALStream.cpp

```diff
@@ -201,7 +201,7 @@ void OpenALStream::SoundLoop()
 		// Convert the samples from short to float
 		float dest[OAL_MAX_SAMPLES * STEREO_CHANNELS];
 		for (u32 i = 0; i < numSamples * STEREO_CHANNELS; ++i)
-			dest[i] = (float)realtimeBuffer[i] / (1 << 16);
+			dest[i] = (float)realtimeBuffer[i] / (1 << 15);
```

Comment:
> Edit: Oops, forgot about the sign, it is correct, but it still seems like we would need to subtract one to get the maximum signed value?

Not exactly. 2s complement s16 samples go from -32768 to +32767, so dividing by 0x8000 gives you fp samples in the range -1.0 to +0.999969482422, while dividing by 0x7fff gives you the range -1.00003051851 to +1.0.